### PR TITLE
Mae cambie y borre World

### PR DESCRIPTION
**Pull request by ThatDudeCR01**

### CODE REVIEW

File: src/App.tsx
Comment (happyhashpant) [inline src/App.tsx:1]: Why did you change the hello world page?

**Review by happyhashpant — CHANGES_REQUESTED** [review]

File: src/App.tsx
Comment (ThatDudeCR01) [reply src/App.tsx:1]: Dude idk

**Review by happyhashpant — APPROVED** [review]
